lib/vmpooler/api/v1.rb

```diff
@@ -289,6 +293,32 @@ def update_user_metrics(operation, vmname)
       puts 'd', "[!] [#{poolname}] failed while evaluating usage labels on '#{vmname}' with an error: #{e}"
     end
 
+    def reset_pool_size(poolname)
+      result = { 'ok' => false }
+
+      pool_index = pool_index(pools)
+
+      pools_updated = 0
+      sync_pool_sizes
+
+      pool_size_now = pools[pool_index[poolname]]['size']
+      pool_size_original = pools_at_startup[pool_index[poolname]]['size']
+      result['pool_size_before_reset'] = pool_size_now
+      result['pool_size_before_overrides'] = pool_size_original
+
+      unless pool_size_now.to_i == pool_size_original.to_i
```

Comment:
One suggestion is to consider specifying 'to_i' when assigning pool_size_now and pool_size_original. Then, comparison does not have to specify to_i and simplifies reading that block.

Reply:
Good call